app/models/tax_return.rb

```diff
@@ -143,6 +143,16 @@ def advance_to(new_status)
     update!(status: new_status) if TaxReturn.statuses[status.to_sym] < TaxReturn.statuses[new_status.to_sym]
   end
 
+  def record_expected_payments!
+    raise StandardError, "Cannot record payments on tax return that is not accepted" unless status == "file_accepted"
+
+    create_accepted_tax_return_analytics!(
```

Comment:
money is an odd value, because they're technically decimal values, but they're decimal values that are always out of 100 and should never "round" the way that typical decimal values do.

By storing as cents, you're able to store the money value as an integer and then convert it to dollars for display values, which helps avoid peksy rounding errors when doing money calculations.

this is a pretty good resource for understanding why we do this from a computer science and engineering POV: https://blog.agentrisk.com/you-better-work-in-cents-not-dollars-2edb52cdf308

The reason we haven't run into this that much is that we actually don't store that many money values right now, and when we do they've typically been whole values.

With this table, if we intend to store refund data from returns that are not always whole number refunds (really anything but stimulus only, like these are), we'd need to store parts of the whole for accuracy.